lib/circuitry/provisioning/queue_creator.rb

```diff
@@ -7,16 +7,23 @@ class QueueCreator
       include Services::SQS
 
       attr_reader :queue_name
+      attr_reader :visibility_timeout
 
-      def self.find_or_create(queue_name, dead_letter_queue_name: nil, max_receive_count: 8)
-        creator = new(queue_name)
+      def self.find_or_create(
+        queue_name,
+        dead_letter_queue_name: nil,
+        max_receive_count: 8,
+        visibility_timeout: 30 * 60
+      )
```

Comment:
:neckbeard: Maybe just me, but the way this method is broken across lines reads difficult.  If you want to keep it multiline, what about this?

```ruby
def self.find_or_create(queue_name,
                        dead_letter_queue_name: nil,
                        max_receive_count: 8,
                        visibility_timeout: 30 * 60)
```